woocommerce/woocommerce
# Update/breadcrumb for accessibility

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Breadcrumb template is missing `aria-label="breadcrumbs"` This PR adds that to element



Closes #40428 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you are using theme that does not have any modifications to WooCommerce breadcrumb template. For example Twenty Fourteen
2. Navigate to archive or product page and inspect breadcrumb element



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
